load.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+// phpcs:ignore
```

Comment:
What's the error in this file for phpcs?

Reply:
```
Namespaced classes and functions should live inside an inc directory. (HM.Files.NamespaceDirectoryName.NoIncDirectory)
```

Reply:
Hmm I thought this was fixed in the latest version of hm-linter 🤔

Reply:
Oh I was getting confused with https://github.com/humanmade/coding-standards/pull/131.

However, there is no defined named functions or classes in file, so what's that all about? cc @rmccue

Reply:
It's probably picking up the inline function as a definition; 131 would fix it anyway.

Reply:
Would y'all have me remove the ignore and allow linting to fail, or should I manually specify the updated coding standards version?

Reply:
hrmmm can we just disable that one rule for this file rather than excluding all code checks? That would be ideal here I think. Sorry for the hold up here.

Reply:
No worries! `// phpcs:ignore` only ignores errors for the line it's defined on or the one right below it, so the rest of the file will still be linted!